static/js/components/dialogs/CollectionFormDialog_test.js

```diff
@@ -197,11 +197,12 @@ describe("CollectionFormDialog", () => {
         })
 
         const expectedRequestPayload = {
-          title:         "new title",
-          description:   "new description",
-          view_lists:    expectedListRequestData,
-          admin_lists:   expectedListRequestData,
-          edx_course_id: "edx-course-id"
+          title:             "new title",
+          description:       "new description",
+          view_lists:        expectedListRequestData,
+          admin_lists:       expectedListRequestData,
+          edx_course_id:     "edx-course-id",
+          is_logged_in_only: false
```

Comment:
Can you add another set of conditions to[ this test here](https://github.com/mitodl/odl-video-service/blob/4bfc90a169638463f323950023cf4e24e114232b/static/js/components/dialogs/CollectionFormDialog_test.js#L99):
```javascript
        [
          "#view-perms-only-logged-in-users",
          "viewChoice",
          SET_VIEW_CHOICE,
          PERM_CHOICE_LOGGED_IN
        ],
```